tinymce-per-block/src/blocks/image-block/index.js

```diff
@@ -72,5 +72,13 @@ registerBlock( 'image', {
 			attrs: { /* caption: block.caption ,*/ align: block.align },
 			rawContent
 		};
+	},
+	create: () => {
+		return {
```

Comment:
I'm not sure yet. I thought about it, but there's also the `transform` case, which I think is a better candidate to be merged with the `create`. We can see the `create` as a special case of transform with an empty content. 

By transform, I mean switching block types, And I was thinking about like:

```js
const rawContent = oldBlock.content; // (some way to extract content from a block)
const newBlock = newBlockDefinition.create( rawContent )
```

What do you think?